### PR TITLE
SAN-23: Update penalty details

### PR DESCRIPTION
**Pull request by acallaghan-ch**
Update penalty details screen

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java [hunk 1]
```diff
@@ -51,6 +53,9 @@ public class EnterPPSDetailsController extends BaseController {
     private static final String ENTER_LFP_DETAILS_MODEL_ATTR = "enterLFPDetails";
```
Comment (klai-ch) [inline src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java:53]: Should we keep this as LFP or we need to change this to PPS?

File: src/main/java/uk/gov/companieshouse/web/pps/controller/pps/NoPenaltyFoundController.java
Comment (rbull-ch) [inline src/main/java/uk/gov/companieshouse/web/pps/controller/pps/NoPenaltyFoundController.java:1]: Did you mean to delete the 2 `NoPenaltyFoundController` files?

File: src/main/resources/templates/pps/details.html [hunk 2]
```diff
@@ -6,51 +6,69 @@
...
     </title>
 </head>
 
     <div id="lfp-enter-details"  layout:fragment="content">
         <form id="lfpenterdetails" th:action="@{''}" th:object="${enterLFPDetails}" method="post" class="form">
+
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-two-thirds">
 
                     <div th:replace="fragments/lfpGlobalErrors :: globalErrors"></div>
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/details.html:19]: I think this should be changed to use the new "There is a problem" alternative introduced recently by @klai-ch 
See `templates/fragments/ppsGlobalErrors.html`?

Comment (acallaghan-ch) [reply src/main/resources/templates/pps/details.html:19]: Missed that, will updated.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -17,4 +17,6 @@ management.endpoints.web.path-mapping.health=late-filing-penalty/healthcheck
 management.endpoint.health.show-details=never
 management.endpoint.health.enabled=true
 management.health.db.enabled=false
-management.health.mongo.enabled=false
\ No newline at end of file
+management.health.mongo.enabled=false
+
+penalty.search.notFound=The details you have entered do not match the information we hold. Enter the company number and penalty reference exactly as they are shown on your penalty notice.
```
[the remark this answers is not in the document]
Comment (rbull-ch) [reply src/main/resources/application.properties:22]: Maybe `penalty.error-details-not-found`?

File: src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java (hunk 1 above)
Comment (acallaghan-ch) [reply src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java:53]: Hmmm, good question Kenneth. I was leaning towards leaving it and then when we do some actual logic refactoring we could do it. i think that is why we left it out of the initial refactor.

File: src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java
Comment (rbull-ch) [inline src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java:1]: I personally think we should drop the PPS from this file name - can we simply rename this to be `EnterDetailsController` for now?

File: src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java (hunk 1 above)
Comment (rbull-ch) [reply src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java:53]: I think if we take away the mix of confusion with having both LFP and PPS here - this version could be simplified to be just `enterDetails` throughout controller / model / template?

Comment (acallaghan-ch) [reply src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java:53]: I didn't really see this as a refactoring ticker per se, but it does make sense to do that

File: src/main/resources/application.properties (hunk 3 above)
[the remark this answers is not in the document]
Comment (acallaghan-ch) [reply src/main/resources/application.properties]: Can do

File: src/main/resources/templates/fragments/lfpGlobalErrors.html
Comment (rbull-ch) [inline src/main/resources/templates/fragments/lfpGlobalErrors.html:1]: Please delete this file as this was only used in the old `details` version?

**Review by rbull-ch — APPROVED** [review]

**Review by dgordon-ch — COMMENTED** [review]
"Approved" with the caveat that we need to look at the image in AC6 on the ticket and how to implement this (as issues with getting it working using thymeleaf).

Also, depending on outcome of investigation (28/11) we can hang fire on merging this for now.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by dgordon-ch — APPROVED** [review]

**Comment by klai-ch** [comment]
Updated code for AC6 part in order to show the image

**Review by klyttle-ch — APPROVED** [review]

File: src/main/resources/templates/pps/details.html [hunk 4]
```diff
@@ -6,51 +6,69 @@
...
 <head>
     <title>
-        Tell us your penalty details
+        Penalty Details
     </title>
 </head>
 
     <div id="lfp-enter-details"  layout:fragment="content">
         <form id="lfpenterdetails" th:action="@{''}" th:object="${enterLFPDetails}" method="post" class="form">
+
```
[the remark this answers is not in the document]
Comment (klai-ch) [reply src/main/resources/templates/pps/details.html]: Update from review suggested

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**